document: some minor fix/update (#216)
chaos-mesh/website: docs/simulate-jvm-application-chaos.md

This document describes how to use Chaos Mesh to create the above fault types of JVM experiments.

> ::: note

Reviewer: ```suggestion
:::note
```

> Make sure your Linux kernel must be v4.1 or later.

Reviewer: ```suggestion
Make sure your Linux kernel is v4.1 or a later version.
```

Author: 需要在前面加上 `Make sure` 吗

> Make sure your Linux kernel must be v4.1 or later.

Reviewer: ```suggestion
Your Linux kernel must be v4.1 or later versions.
```
